extensions/cornerstone/src/ConnectedCornerstoneViewport.js

```diff
@@ -23,36 +23,41 @@ const MEASUREMENT_ACTION_MAP = {
 const mapStateToProps = (state, ownProps) => {
   let dataFromStore;
 
+  // TODO: This may not be updated anymore :thinking:
   if (state.extensions && state.extensions.cornerstone) {
     dataFromStore = state.extensions.cornerstone;
   }
 
-  // TODO: This should be extension configuration
-  // ...dataFromStore -->
-  // availableTools,
-  // onNewImage,
-  // onRightClick,
-  // onTouchPress,
-  // onTouchStart,
-  // onMouseClick,
-
   // If this is the active viewport, enable prefetching.
   const { viewportIndex } = ownProps; //.viewportData;
   const isActive = viewportIndex === state.viewports.activeViewportIndex;
+  const isStackPrefetchEnabled = isActive;
   const viewportSpecificData =
     state.viewports.viewportSpecificData[viewportIndex] || {};
 
+  // CINE
+  let isPlaying = false;
+  let frameRate = 24;
+
+  if (viewportSpecificData && viewportSpecificData.cine) {
```

Comment:
This isn't this PR's fault I think, but I just noticed the CINE dialog is no longer draggable, is that a known issue?

Reply:
Shot you a note. It _was_ working, but stopped after I force updated our yarn lock. I think this is a loose versioning issue. Will hold on merging until resolved.